manifests/rules/docker_ce.pp

```diff
@@ -0,0 +1,111 @@
+# @summary Default firewall configuration for Docker-CE
+#
+# The configuration distributed in this class represents the default firewall
+# configuration done by docker-ce when the iptables integration is enabled.
+#
+# This class is needed as the default docker-ce rules added to ip-filter conflict
+# with the inet-filter forward rules set by default in this module.
+#
+# When using this class 'docker::iptables: false' should be set.
+#
+# @param docker_interface
+#   Interface name used by docker. It defaults to docker0.
+# @param docker_prefix
+#   The address space used by docker. It defaults to 172.17.0.0/16.
```

Comment:
```suggestion
#   The address space used by docker.
```
The default value will be automatically picked up from the code when generating `REFERENCE.md` so there's no need to specify it here again.

Reply:
Done.